lib/osf-components/addon/components/registries/schema-block-group-renderer/component.ts

```diff
@@ -0,0 +1,45 @@
+import { tagName } from '@ember-decorators/component';
+import { computed } from '@ember-decorators/object';
+import Component from '@ember/component';
+
+// import Changeset from 'ember-changeset';
+
+import { layout } from 'ember-osf-web/decorators/component';
+import { SchemaBlock, SchemaBlockGroup } from 'ember-osf-web/packages/registration-schema';
+
+import template from './template';
+
+@layout(template)
+@tagName('')
+export default class SchemaBlockGroupRenderer extends Component {
+    // Required parameters
+    schemaBlockGroup!: SchemaBlockGroup;
+    // changeset!: Changeset;
```

Comment:
Maybe uncomment this to make it clear that this is a required parameter for this component?